src/database/migrations/20240531070905-alter-resource-add-title.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		// Adding a new column to the 'resources' table
+		await queryInterface.addColumn('resources', 'title', {
+			type: Sequelize.STRING,
+			allowNull: true,
+		})
```

Comment:
@adithyadinesh0412 add index to title
await queryInterface.addIndex('resources', ['title']);